autogpt_platform/backend/backend/executor/manager.py

```diff
@@ -1120,6 +1145,74 @@ def _validate_node_input_credentials(self, graph: GraphModel, user_id: str):
                         "type/provider mismatch"
                     )
 
+    async def _moderate_graph_content(
```

Comment:
Can we make this a public method and place this outside the manager.py? We can putit in iffy folder
There will be an incoming refactor of this file so better to avoid more code conflict.